core/templates/pages/learner-dashboard-page/goals-tab.component.ts

```diff
@@ -206,7 +209,7 @@ export class GoalsTabComponent implements OnInit {
       targetElement &&
       !this.dropdownRef.nativeElement.contains(targetElement)
     ) {
-      this.showThreeDotsDropdown = false;
+      this.showThreeDotsDropdown[i] = false;
```

Comment:
I think you would have to run a loop (from 0 (inclusive) to the total number of goals (not inclusive)) and set them all to false.

Reply:
Done!